timescaledb/tutorials/analyze-intraday-stocks/explore-stocks-data.md

```diff
@@ -0,0 +1,287 @@
+    GROUP BY symbol
+    ORDER BY volume DESC
+    LIMIT 5
+""".format(bucket="14 day")
+df = pd.read_sql(query, conn)
+fig = px.bar(df, x='symbol', y='volume', title="Most traded symbols in the last 14 days")
+fig.show()
+```
+
+![most traded symbols](https://assets.timescale.com/docs/images/tutorials/intraday-stock-analysis/most_traded_symbols.png)
+
+### 2. How did Apple's trading volume change over time?
+
+Now let's try a similar query focused on the daily trading volume of one symbol (e.g. 'AAPL'). 
+
+```python
+query = """
+    SELECT time_bucket('{bucket}', time) AS bucket, sum(trading_volume) AS volume
+    FROM stocks_intraday
+    WHERE symbol = '{symbol}'
+    GROUP BY bucket
+    ORDER BY bucket
+""".format(bucket="1 day", symbol="AAPL")
+df = pd.read_sql(query, conn)
+fig = px.line(df, x='bucket', y='volume', title="Apple's daily trading volume over time")
+fig.show()
+```
+
+![apple trading volume over time](https://assets.timescale.com/docs/images/tutorials/intraday-stock-analysis/apple_trading_volume.png)
+
+### 3. How did Apple's stock price change over time?
+
+This query returns the weekly stock price of Apple over time:
+
+```python
+query = """
+    SELECT time_bucket('{bucket}', time) AS bucket,
+    last(price_close, time) AS last_closing_price
+    FROM stocks_intraday
+    WHERE symbol = '{symbol}'
+    GROUP BY bucket
+    ORDER BY bucket
+""".format(bucket="7 days", symbol="AAPL")
+df = pd.read_sql(query, conn)
+fig = px.line(df, x='bucket', y='last_closing_price')
+fig.show()
+```
+
+![apple price over time](https://assets.timescale.com/docs/images/tutorials/intraday-stock-analysis/apple_price.png)
+
+
+### 4. Which symbols had the highest weekly gains?
+
+Now generate a table containing the symbols with the biggest weekly gains:
+
+```python
+query = """
+    SELECT symbol, bucket, max((closing_price-opening_price)/closing_price*100) AS price_change_pct
+    FROM ( 
+        SELECT 
+        symbol, 
+        time_bucket('{bucket}', time) AS bucket, 
+        first(price_open, time) AS opening_price, 
+        last(price_close, time) AS closing_price
+        FROM stocks_intraday
+        GROUP BY bucket, symbol
+    ) s
+    GROUP BY symbol, s.bucket
+    ORDER BY price_change_pct {orderby}
+    LIMIT 5
+""".format(bucket="7 days", orderby="DESC")
+df = pd.read_sql(query, conn)
+print(df)
+```
+
+|symbol |bucket     |price_change_pct |
+|-------|-----------|-----------------|
+|ZM     |2021-06-07 |24.586495        |
+|TSLA   |2021-01-04 |18.280314        |
+|BA     |2021-03-08 |17.745225        |
+|SNAP   |2021-02-01 |16.149649        |
+|TSLA   |2021-03-08 |15.842941        |
+
+`price_change_pct` shows the price change that happened between the start and end of the week.
+
+`bucket` shows (the first day of) the week.
+
+<highlight type="tip">
+Change `orderby` to "ASC" to query the biggest losses.
+</highlight>
+
+### 5. Weekly FAANG prices over time?
+
+Let's see a line chart with the FAANG weekly stock prices:
```

Comment:
I feel like it's worth spelling out somewhere what FAANG is. I mean, I guess if people are going to do a stock trading app they probably know the big 5 (and I realize it's in the query... but in a different order...). Anyway, just a thought.

Reply:
https://github.com/timescale/docs/pull/150/commits/8eb4e978a2e666b407e839672db56e44d16c7322